src/drt/src/db/obj/frNet.h

```diff
@@ -50,9 +51,10 @@ class frNet : public frBlockObject
 {
  public:
   // constructors
-  frNet(const frString& in)
+  frNet()
```

Comment:
If you can construct an `frNet` with no name and no `dbNet*` then how can you disambiguate two empty `frNet`s?

If this is to support fake nets then I would suggest removing the default constructor and instead having a `frNet(bool, bool)` constructor to specify `isFakeVSSNet_` and `isFakeVDDNet_`.